WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfoTabViewModel.cs

```diff
@@ -7,10 +12,23 @@ public class TransactionInfoTabViewModel : WasabiDocumentTabViewModel
 	{
 		public TransactionInfoTabViewModel(TransactionDetailsViewModel transaction) : base("")
 		{
+			Global = Locator.Current.GetService<Global>();
```

Comment:
It would be nicer to get the dependency you need via constructor - namely `UiConfig`.

Reply:
Do you mean to get the `UiConfig` instead of `Global` from the locator?

Reply:
I mean: Not to use Locator at all. Just to pass the needed dependency via constructor so that the code is testable. 

It may not be straightforward but it's certainly good thing to do.

Reply:
Done, but it is kinda strange to me because I had to make changes in two constructors to achieve this, and Locator was used also where `Global` is founded first.

Reply:
`Locator` is used everywhere and it's not a good thing - because of tests & complexity. There were various PRs regarding `Locator` https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Apr+locator

Using `Global` makes you code shorter but when you have a class where you pass dependencies via constructor, it typically makes you think more about design of the class and thus you avoid extremely complex classes everyone is afraid to touch (not this case). It's a long term investment though.